# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pciutils/zlib.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

libpci depends on zlib, and -lz has to be added to the linker command
line after libpci.a (it is incorrectly passed in LDFLAGS now)  

--- pciutils-2.2.7/Makefile.orig        2007-10-05 14:22:06.000000000 +0200
+++ pciutils-2.2.7/Makefile     2007-10-10 12:34:55.000000000 +0200
@@ -19,7 +19,7 @@
 DIRINSTALL=install -d
 AR=ar
 RANLIB=ranlib
-PCILIB=lib/libpci.a
+PCILIB=lib/libpci.a -lz
 PCILIBPC=lib/libpci.pc
 PCIINC=lib/config.h lib/header.h lib/pci.h lib/types.h lib/sysdep.h
 PCIINC_INS=lib/config.h lib/header.h lib/pci.h lib/types.h
